core/store/src/db.rs

```diff
@@ -49,22 +49,66 @@ impl Into<io::Error> for DBError {
 #[derive(PartialEq, Debug, Copy, Clone, EnumIter, BorshDeserialize, BorshSerialize, Hash, Eq)]
```

Comment:
Let's maybe add comment for the enum as a whole (or maybe even `//!` comment at the start of the file) which explains terminology here. Ie, what are columns and rows? I haven't worked with key-value stores a lot, so it is a *bit* confusing to me.

Reply:
I've added some explanation on top of DBCol - ptal